lua/ttt2/libraries/entspawn.lua

```diff
@@ -17,8 +17,65 @@ local mathMin = math.min
 local timerCreate = timer.Create
 local timerRemove = timer.Remove
 
+local allowForcedRandomSpawn = false
+
 entspawn = entspawn or {}
 
+---
+-- Enable or disable forced random spawns for 'env_entity_maker' 'https://developer.valvesoftware.com/wiki/Env_entity_maker'
+-- spawning non available random spawns at map start
+-- @param bool enable The state to set it to
+-- @realm server
+function entspawn.EnableForcedRandomSpawn(enable)
```

Comment:
```suggestion
function entspawn.SetForcedRandomSpawn(enable)
```
A function that has a parameter should not be named after one of the states, that is often confusing.

Reply:
ah makes sense, I wondered how to describe it, as its not a toggle nor a direct enabler. Set sounds good.